SUPPORT.md

```diff
@@ -0,0 +1,9 @@
+# Support
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please [search](https://github.com/Azure/azure-service-operator/issues) the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as [a new issue](https://github.com/Azure/azure-service-operator/issues/new/choose). The Azure Service Operator project maintainers will respond to the best of their abilities.
```

Comment:
Should we remove this information from the read me (and replace with a section linking here?)

Just reducing document duplication?

Also I see the README says:
> Azure Service Operator is an open source project that is **not** covered by the [Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). 

Should be include that here too?

Reply:
I've added that link to `SUPPORT.md`.

Won't remove the duplication from the `README.md` as the goal is to make support details more discoverable, not to require people to look in one specific place.